scamp/scamp-3.c

```diff
@@ -1244,9 +1245,9 @@ void proc_100hz (uint a1, uint a2)
                 }
             }
         }
-      else if (netinit_tick_counter >= 3)
+      else if (netinit_biff_tick_counter >= 20)
```

Comment:
3 should be sufficient: You lose up to one tick of timer accuracy to timer speed differences and you lose up to one additional tick of timer accuracy to timer phase differences.

Have you been experiencing issues with 3?